translations/es.ts

```diff
@@ -1215,7 +1215,7 @@ serán enviados cuando los veas aparecer de nuevo en línea.</translation>
     <message>
         <location filename="../src/widget/form/settings/generalsettings.ui" line="521"/>
         <source>Timestamp format:</source>
-        <translation>Formato de marca temporal:</translation>
+        <translation>Formato de marcas temporales:</translation>
```

Comment:
> pero seguro que vos no estás de la mismo opinión
jajaja, en un primer momento sí, pero estoy de acuerdo con "la fecha".

@zetok aaannndddyyy is right: `Timestamp format` refers actually only to the date. Changing it to `Date format` would make it more understandable for the non-technical users. Should I open a new issue?